app/src/main/java/com/example/mytrainingpal/screens/Calendar.kt

```diff
@@ -56,10 +60,13 @@ fun CalendarScreenContent() {
                     )
                 }
             })
+        if(allExercises.isNotEmpty()){
         LazyColumn(modifier = Modifier.fillMaxWidth()) {
-            items(placeholderNumbers) { number ->
-                Greeting(name = "$number")
+            items(allExercises) { exercise ->
+                Greeting(name = exercise.name)
             }
```

Comment:
Is there a way to make it so that AndroidStudio also recognizes the Type of `exercise` ? Because it does not seem to realize the attributes of it otherwise.

Reply:
I don't know why `AndroidStudio` doesn't recognizes the Type of `exercise` 🤷 . Maybe just restart the IDE